src/Common/SurfaceFluxes/SurfaceFluxes.jl

```diff
@@ -271,6 +272,64 @@ function compute_physical_scale(
     return _π_group⁻¹ * _von_karman_const / Σterms * (x_in - x_s)
 end
 
+"""
+    recover_profile(z, x_star, x_s, z_0, L_MO, transport, ::DGScheme, uf)
+
+ - `z` Input height for evaluation
+ - `x_star` Surface layer scales for state variable array `x`.
+ - `x_s` surface values for state variable array `x`
+ - `z_0` roughness lengths for state variable array `x`
+ - `L_MO` Obukhov length
+ - `transport` Type of transport (momentum or heat)
+ - `uf` universal function family
+
+    Recover vertical profiles u(z), θ(z), ... using equations (4) and (5)
+    for DG and (12), (13) for FV from Nishizawa & Kitamura (2018).
+"""
+function recover_profile(
```

Comment:
If we're exporting `recover_profile`, and knowing how much `monin_obukhov_length` will likely be used, can we write a method `recover_profile(:: SurfaceFluxConditions)` and `monin_obukhov_length(:: SurfaceFluxConditions)` so that users are encouraged to use a simpler interface? It might be easier, even if we need to add more properties to `SurfaceFluxConditions`.

Reply:
The SurfaceFluxConditions contains a Monin Obukhov Length, this is kind of a chicken and egg situation.

Reply:
And the SurfaceFluxConditions are built with the iterative solve. Both `monin_obuhov_length` and recover_profile are not iterative. In addition, the output of `recover_profile` is an input in `surface_conditions`.

Reply:
Oh, I see. It's more acting as a helper for constructing the input. This is just for the ICs, though, right? It might still be a good idea to define
```julia
monin_obukhov_length(sc::SurfaceConditions) = sc.L_MO
```
In case we're able to eventually hide the ICs from the user. Maybe it'll be easier to know if this is useful after seeing it used, though. Up to you.

Reply:
I agree this is a good idea, which we can probably use in the EDMF closures. I will add this getter function.